tests/sql/timestamp.sql

```diff
@@ -0,0 +1,49 @@
+create table timestamp_test
+(
+    ts timestamp
+);
+insert into timestamp_test
+values ('1991-01-08 04:05:06');
+insert into timestamp_test
+values ('1991-01-09 04:05:06 AD');
+insert into timestamp_test
+values ('1991-01-10 04:05:06 BC');
+insert into timestamp_test
+values ('1991-01-14 04:05:06 +08:00');
+insert into timestamp_test
+values ('1991-01-15 04:05:06 AD +08:00');
+insert into timestamp_test
+values ('1991-01-16 04:05:06 BC +08:00');
+insert into timestamp_test
+values ('1991-01-17 04:05:06 +08:00 AD');
+insert into timestamp_test
+values ('1991-01-18 04:05:06 +08:00 BC');
+select *

```

Comment:
IMHO, it might be better to use *.slt format, which can reflect query results. Here is E2E integration test guide https://github.com/risinglightdb/risinglight/blob/main/docs/05-e2e-tests.md

Reply:
Thanks fot pointing out this.I am tring to solve it but meet an error as the ci shows.Do you have any idea about this?

Reply:
close #767

Reply:
Also it might be better to add `close #767 ` to https://github.com/risinglightdb/risinglight/pull/769#issue-1533115570, which can help close a issue